platforms-whitepaper/v1alpha1/paper.md

```diff
@@ -154,62 +154,55 @@ of capabilities, platforms make it easy for their users to deliver valuable prod
-and use cases. It serves those requirements through consistent interfaces of
-several types \[3\], particularly HTTP APIs, Web portals, and command-line tools.
-For example, all platform users require the ability to observe their workloads,
-so a platform might offer a capability to collect and display logs, metrics and
-traces from applications. It might enable users to request and use that
-capability via an API and a Web page.
-
-To be able to use a platform's offerings, users require documentation and
-examples \[4\]. So a platform team needs to not only make a capability like
-observability available, but also publish templates and docs to help users
-consume it. For example, a platform team might publish a reusable workflow for
-building, testing, deploying and verifying a Web application on Kubernetes.
-Beyond examples for individual resource types, advanced platforms should group
-collections of templates and guidance for specific scenarios, a bundle often
-described as a golden path.
-
-The capabilities provided by a platform should be available to users on demand
-\[5\] with minimal manual intervention. The interfaces discussed above - APIs and
-web pages - should enable users to get resources they need in a few minutes with
-at most an approval. For example, a web page describing a database management
-service should include a button to request a database and share a locator and
-credentials for that database within a few minutes of the request.
-
-One of the purposes of a platform is to reduce cognitive load on product teams,
-so services offered by the platform should not need to be operated by consumers;
-either the platform provider or its delegates, such as a cloud service, should
-handle those operations, hiding complexity \[7\] from users. For example, users
-may often require a message queue, but shouldn't have to manage a message
-broker.
-
-Platforms are intended to make product development more efficient, so it's
-important they not be an impediment \[8\] when the most efficient path is to not
-use the platform's offerings. Platforms should enable product teams to
-provide and manage their own capabilities when necessary. For example, if a
-platform team doesn't provide a graph database and it's required for a product,
-it should be possible for that product team to provision and operate a graph
-database themselves.
+After defining what a platform is and why an organization might want to build one,
+it's relevant to identify what are the key attributes that affect the success
+of a platform.
+
+1. **Platform as a product**. A platform exists to serve the requirements of its users
+   and it should be designed and evolved based on those requirements, similar to any
+   other software products.
+1. **Common requirements**. A platform should provide the necessary capabilities to
+   support the most common use cases across stream-aligned teams, and prioritize those
```

Comment:
> stream-aligned teams

Do we define this anywhere? I love Team Topologies, but this is not a globally known concept (yet!).

*Note: Of course this is a question for the paper and not this specific PR, just want to call attention as I spotted it here!

Reply:
Do we feel that the wording is so self-explanatory that we could use it even if the reader doesn't know Team Topologies?

Reply:
Good point! I wasn't sure about this, and then I saw we were adding the definition in https://github.com/cncf/tag-app-delivery/pull/298/files and went with "stream-aligned teams". But I agree that "product teams" would be better in this context since it's more self-explanatory. I'll change that.

Reply:
I have replaced "stream-aligned teams" with "product teams".